deepcell_tracking/utils_test.py

```diff
@@ -161,6 +162,17 @@ def test_save_trks(self, tmpdir):
         np.testing.assert_array_equal(X, loaded['X'])
         np.testing.assert_array_equal(y, loaded['y'])
 
+        # test save trks to bytes
+        b = io.BytesIO()
+        out = utils.save_trks(b, lineage, X, y)
+        assert isinstance(out, io.BytesIO)
```

Comment:
As mentioned above, I don't think its necessary to return anything from `save_trks`. (I'm pretty sure `b == out` right here).

Saving and loading to/from the bytes object is enough.